src/libui_sdl/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ if (WIN32)
 endif()
 
 option(BUILD_SHARED_LIBS "Whether to build libui as a shared library or a static library" ON)
-set(BUILD_SHARED_LIBS OFF)
+set(BUILD_SHARED_LIBS ON)
```

Comment:
This saves the trouble of having to configure a static SDL2 lib in CI.

I should probably put this behind a compiler flag so it won't break your workflow.